### ISSUE TITLE
Navigating through the user interface using tabbing and arrow keys is difficult.

### ISSUE THREAD

**Issue by msftedad** [microsoft/vsts-extension-retrospectives#1204]
#A11y #accessibility

**Describe the bug**
Navigating through the user interface using tabbing and arrow keys is difficult. Users relying on keyboard navigation face challenges in efficiently moving through the UI, which significantly affects accessibility and usability.

**Are you using the Retrospectives Tool on-prem?**
No

**To Reproduce**
1. open the URL https://dev.azure.com/eDADPlayground/SCRUMProject .
2. Expand left navigation and select boards-->Retrospectives.
3. Now under the 'board' tab we have different feedback groups.
4. verify whether navigating through the user interface using the Tab key and arrow keys is difficult or not.

**Expected behavior**
Users should be able to navigate seamlessly and efficiently through the UI using the Tab key and arrow keys.
 

**Screenshots**

https://github.com/user-attachments/assets/2f45d42b-40d1-4d68-9688-6d4b95de8e0b

<img width="960" alt="Image" src="https://github.com/user-attachments/assets/50ca9ad9-3917-4788-ac28-1e75a2e26f92" />

**What screen type are you seeing this on ?**
 Desktop Windows 11

**Additional context**
